README.md

```diff
@@ -7,7 +7,7 @@ gathered by the Elastic [Beats](https://www.elastic.co/products/beats).
 Installation
 -------------
 
-To load the dashboards, execute the script pointing to the Elasticsearch HTTP
+To load the dashboards execute the script pointing to the Elasticsearch HTTP
```

Comment:
The original sentence is grammatically correct. A comma usually follows introductory phrases of four or more words.